app/models/college.rb

```diff
@@ -0,0 +1,9 @@
+class College < ActiveRecord::Base
+  
+  #find the conference of a college   
+  def find_conference_id
```

Comment:
This method seems like it works just fine, but it's named inaccurately. It should be `find_conference_name`.